src/tiktokbot.ts

```diff
@@ -0,0 +1,283 @@
+     * @return object
+     * 
+     * @example 
+     * <TikTok>.reply: {...} (object)
+     */
+    private get reply() {
+        return { content: null, embeds: [], files: [], components: [], allowedMentions: { repliedUser: false } }
+    }
+
+    /**
+     * Default error catching action.
+     * 
+     * @param [] None required.
+     * @return Function
+     * 
+     * @example 
+     * <TikTok>.default: () => null (Function)
+     */
+    private get default() {
+        return () => null
+    }
+
+    /**
+     * Merge multiple objects into one.
+     * 
+     * @param [main] The main object to merge into.
+     * @param [secondary] The sub-objects to merge into the main.
+     * @return object
+     * 
+     * @example
+     * <TikTok>.mergify({ a: 1 }, { b: 2 }): { a: 1, b: 2 } (object)
+     */
+    private mergify = (main: { [key: string]: any }, ...secondary: object[]) => {
+        secondary.map((obj: { [key: string]: any }) => Object.keys(obj).map((k: string) => main[k] = obj[k]))
+        return main
+    }
+
+    /**
+     * Make a millisecond time number readable.
+     * 
+     * @param [ms] The ms time to convert.
+     * @return string
+     * 
+     * @example
+     * <TikTok>.timeify(60 * 1000): "1 min(s)" (string)
+     */
+    private timeify = (ms: number) => {
+        // Declare variables.
+        let sec: number = Math.round(ms / 1000)
+        let min: number = Math.floor(sec / 60)
+        let hrs: number = 0
+        const final: string[] = []
+
+        // Complete logic.
+        if (min >= 60) { (hrs = Math.floor(min / 60)) ? final.push(`${hrs} hr(s)`) : null }
+        { (min -= hrs * 60) ? final.push(`${min} min(s)`) : null }
+        { (sec = Math.floor(sec % 60)) ? final.push(`${sec} sec(s)`) : null }
+        return final.join(', ')
+    }
+
+    /**
+     * <discord.js> Client event: "ready".
+     * 
+     * @param [] None required.
+     * @return null
+     * 
+     * @example 
+     * <TikTok>.ready(): null
+     */
+    private ready = () => console.log(`Connected to ${this.user?.tag ?? 'Client'}.`)
+
+    /**
+     * <discord.js> Client event: "messageCreate".
+     * 
+     * @param {import('discord.js').Message} [message] <discord.js> Message object.
+     * @return null
+     * 
+     * @example 
+     * <TikTok>.messageCreate(<Message>): null
+     */
+    private async messageCreate(message: Message) {
+
+        // Check for bot message or DM message.
+        if (message.author?.bot || message.channel?.type?.is('DM')) return
+
+        // Check for commands.
+        if (message.content.match(new RegExp(`^(<@!?${this.user?.id}>.*)`, 'gm'))?.some((e: string) => e)) {
+
+            // Set custom message properties.
+            { (message as MessageExtender).args = message.content.split(' ').slice(1) }
+            { (message as MessageExtender).command = (message as MessageExtender).args.shift() ?? '' }
+
+            if ((message as MessageExtender).command.is('invite')) {
```

Comment:
Couldn't you just do 
```diff
- messageCreate(message: Message) {
+ messageCreate(message: MessageExtender) {
```
instead of recasting it

Reply:
That will cause an issue within the `<Client>.on('messageCreate', ...)` event.

Reply:
Ah, nvm then. ts can be a dick about that lol